packages/app/lib/models/sightings.dart

```diff
@@ -70,6 +72,17 @@ class Sighting {
       String? comment,
       List<Species>? species,
       List<LocalName>? localNames}) async {
+    Species? updatedSpecie = species?.first;
+    LocalName? updatedLocalName = localNames?.first;
```

Comment:
ah looks like Dart doesn't like this - will need to update this implementation a bit 😅

Reply:
addressed via https://github.com/p2panda/meli/pull/60/commits/302293aa9032b8dbf9cf3a87c2ad439f942a01d0